### PR TITLE
Update originate command to select contract file

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
# 🏗️ PR ➾

Fixes # (issue)

## 🪂 Pre-Merge Checklist

- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🏄‍♂️ Myself and others have built this branch and done manual testing on the change
- [ ] ⛵ A summary describing the change has been written for release notes

----------------------------------------------------------------------------------------------------------------------------

## 🪁 Description

Please include a summary of the change and add the issue number this is related to above. Please also include relevant motivation and context for the change
(e.g. what was the existing behavior, why did it break, etc.)

## 🎢 Test Plan

Please describe the testing strategy for this change

----------------------------------------------------------------------------------------------------------------------------

### 🛸 Type of Change

- [ ] 🐟 Minor change (non-breaking change of very limited scope)
- [ ] 🦑 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🌊 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🐳 Major refactor (breaking changes and significant impact to the codebase)

----------------------------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Review by jchenche — APPROVED** [review]
Awesome!

File: taqueria-vscode-extension/src/lib/helpers.ts [hunk 1]
```diff
@@ -619,13 +618,32 @@ export class VsCodeHelper {
...
+					const artifactsFolderUri = api.Uri.file(artifactsFolder);
+					const file = await this.vscode.window.showOpenDialog({
+						canSelectFiles: true,
+						canSelectFolders: false,
+						canSelectMany: false,
+						defaultUri: artifactsFolderUri,
+						openLabel: 'Originate',
+						title: 'Select contract to originate',
+						filters: {
+							'Contract Files': ['*.tz'],
```
Comment (jchenche) [inline taqueria-vscode-extension/src/lib/helpers.ts:632]: It has to exclude .*\.(default_storage|storage|parameter)

Comment (jchenche) [reply taqueria-vscode-extension/src/lib/helpers.ts:632]: @AlirezaHaghshenas e.g. `hello.storage.someStorageName.tz` is not a contract

**Review by jchenche — APPROVED** [review]

File: taqueria-vscode-extension/src/lib/helpers.ts (hunk 1 above)
Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/src/lib/helpers.ts:632]: That is actually a separate task. I can exclude all storage files both from this dialog, and from the artifacts treeview.
@hu3man.

**Review by sinapsist — APPROVED** [review]
LGTM

**Review by alexzbusko — APPROVED** [review]

**Pull request merged**